test/runtests.jl

```diff
@@ -54,7 +54,9 @@ model = atom_ols
 
 p_distr = predict(atom_ols, fitresult, selectrows(X, test))
 
-@test p_distr[1] == Distributions.Normal(p[1], GLM.dispersion(fitresult.model))
+@test p_distr[1] == Distributions.Normal(
+    p[1], MLJGLMInterface.dispersion(fitresult)
+)
```

Comment:
```suggestion
dispersion =  MLJGLMInterface.dispersion(fitresult)
@test p_distr[1] == Distributions.Normal(p[1], dispersion)
```
It wasn't really readable as multiline